cosmos/infra/azuredeploy_mongo.json

```diff
@@ -461,4 +461,4 @@
             "value": "The Benchmarking job has been triggered successfully. Please check the storage account you provided for Job Status and Results. The jobs status will be available in a storage table within a few minutes and results will be available once the job finishes."
         }
     }
-}
+}
```

Comment:
can you fix this? There should be no change for azuredeploy_mongo.json.

Reply:
Blank line added while reverting the changes.

Now removed.